test/unit/libraries/EnumerableGuardianMap/removeAll.t.sol

```diff
@@ -2,21 +2,82 @@
 pragma solidity ^0.8.25;
 
 import "forge-std/console2.sol";
-import { UnitBase } from "../../UnitBase.t.sol";
+import {UnitBase} from "../../UnitBase.t.sol";
+import {EnumerableGuardianMap, GuardianStorage, GuardianStatus} from "../../../../src/libraries/EnumerableGuardianMap.sol";
 
 contract EnumerableGuardianMap_removeAll_Test is UnitBase {
+    using EnumerableGuardianMap for EnumerableGuardianMap.AddressToGuardianMap;
+    mapping(address account => EnumerableGuardianMap.AddressToGuardianMap guardian)
+        internal guardiansStorage;
+
     function setUp() public override {
         super.setUp();
     }
 
-    function test_RemoveAll_RevertWhen_TooManyValuesToRemove() public view {
-        // TODO: test
+    function test_RemoveAll_Succeeds() public {
+        bool result;
+
+        for (uint256 i = 1; i <= 3; i++) {
+            result = guardiansStorage[accountAddress].set({
+                key: vm.addr(i),
+                value: GuardianStorage(
+                    GuardianStatus.REQUESTED,
+                    guardianWeights[1]
+                )
+            });
+            assertEq(result, true);
+        }
+        address[] memory addresses = new address[](3);
+        addresses[0] = vm.addr(1);
+        addresses[1] = vm.addr(2);
+        addresses[2] = vm.addr(3);
+        guardiansStorage[accountAddress].removeAll(addresses);
     }
 
-    function test_RemoveAll_Succeeds() public view {
-        // TODO: test
+    function test_RemoveAll_RemovesMaxNumberOfValues() public {
+        bool result;
+
+        for (
+            uint256 i = 1;
+            i <= EnumerableGuardianMap.MAX_NUMBER_OF_GUARDIANS;
+            i++
+        ) {
+            result = guardiansStorage[accountAddress].set({
+                key: vm.addr(i),
+                value: GuardianStorage(
+                    GuardianStatus.REQUESTED,
+                    guardianWeights[1]
+                )
+            });
+            assertEq(result, true);
+        }
+
+        address[] memory addresses = new address[](
+            EnumerableGuardianMap.MAX_NUMBER_OF_GUARDIANS
+        );
+        for (
+            uint256 i = 0;
+            i < EnumerableGuardianMap.MAX_NUMBER_OF_GUARDIANS;
+            i++
+        ) {
+            addresses[i] = vm.addr(i + 1);
+        }
+        guardiansStorage[accountAddress].removeAll(addresses);
```

Comment:
Same here, can you add an assert to check that the values are no longer in storage?